iac-extensions/spring-config/src/test/resources/checks/HardcodedSecretsCheck/HardcodedSecretsCheck.properties

```diff
@@ -0,0 +1,232 @@
+spring.neo4j.authentication.password=${PASSWORD}
+spring.r2dbc.password=${PASSWORD}
+spring.flyway.password=${PASSWORD}
+spring.liquibase.password=${PASSWORD}
+spring.sql.init.password=${PASSWORD}
+spring.activemq.password=${PASSWORD}
+spring.artemis.password=${PASSWORD}
+spring.kafka.admin.ssl.key-password=${PASSWORD}
+spring.kafka.admin.ssl.key-store-password=${PASSWORD}
+spring.kafka.admin.ssl.trust-store-password=${PASSWORD}
+spring.kafka.consumer.ssl.key-password=${PASSWORD}
+spring.kafka.consumer.ssl.key-store-key=${PASSWORD}
+spring.kafka.consumer.ssl.key-store-password=${PASSWORD}
+spring.kafka.consumer.ssl.trust-store-password=${PASSWORD}
+spring.kafka.producer.ssl.key-password=${PASSWORD}
+spring.kafka.producer.ssl.key-store-key=${PASSWORD}
+spring.kafka.producer.ssl.key-store-password=${PASSWORD}
+spring.kafka.producer.ssl.trust-store-password=${PASSWORD}
+spring.kafka.ssl.key-password=${PASSWORD}
+spring.kafka.ssl.key-store-password=${PASSWORD}
+spring.kafka.ssl.trust-store-password=${PASSWORD}
+spring.kafka.streams.ssl.key-password=${PASSWORD}
+spring.kafka.streams.ssl.key-store-key=${PASSWORD}
+spring.kafka.streams.ssl.key-store-password=${PASSWORD}
+spring.kafka.streams.ssl.trust-store-password=${PASSWORD}
+spring.rabbitmq.password=${PASSWORD}
+spring.rabbitmq.ssl.key-store-password=${PASSWORD}
+spring.rabbitmq.ssl.trust-store-password=${PASSWORD}
+spring.rabbitmq.stream.password=${PASSWORD}
+server.ssl.key-password=${PASSWORD}
+server.ssl.key-store-password=${PASSWORD}
+server.ssl.trust-store-password=${PASSWORD}
+spring.security.oauth2.resourceserver.opaquetoken.client-secret=${PASSWORD}
+spring.security.user.password=${PASSWORD}
+spring.rsocket.server.ssl.key-password=${PASSWORD}
+spring.rsocket.server.ssl.key-store-password=${PASSWORD}
+spring.rsocket.server.ssl.trust-store-password=${PASSWORD}
+management.appoptics.metrics.export.api-token=${PASSWORD}
+management.datadog.metrics.export.api-key=${PASSWORD}
+management.datadog.metrics.export.application-key=${PASSWORD}
+management.dynatrace.metrics.export.api-token=${PASSWORD}
+management.elastic.metrics.export.api-key-credentials=${PASSWORD}
+management.elastic.metrics.export.password=${PASSWORD}
+management.humio.metrics.export.api-token=${PASSWORD}
+management.influx.metrics.export.password=${PASSWORD}
+management.influx.metrics.export.token=${PASSWORD}
+management.kairos.metrics.export.password=${PASSWORD}
+management.newrelic.metrics.export.api-key=${PASSWORD}
+management.prometheus.metrics.export.pushgateway.password=${PASSWORD}
+management.server.ssl.key-password=${PASSWORD}
+management.server.ssl.key-store-password=${PASSWORD}
+management.server.ssl.trust-store-password=${PASSWORD}
+management.signalfx.metrics.export.access-token=${PASSWORD}
+management.wavefront.api-token=${PASSWORD}
+spring.devtools.remote.secret=${PASSWORD}
+
+#---
+# Noncompliant@+1
+spring.datasource.password=${PASSWORD}-suffix
+#---
+# Noncompliant@+1
+spring.datasource.password=prefix-${PASSWORD}
```

Comment:
Do we actually want to raise here? IMO we should clarify this cases with AppSec.

Reply:
Clarified; for such strings, we shouldn't raise. I'll update the PR.

Reply:
Thanks for clarifying it! It will make it a bit more complicated though